tests/typecheck/models/test_related_fields.yml

```diff
@@ -87,3 +87,38 @@
               b = models.ForeignKey(Model2, related_name="test2", on_delete=models.CASCADE)
 
               objects = Model4Manager()
+
+-   case: test_related_name_foreign_object_multi_column
+    main: |
+      from app1.models import Model1, Model2
+
+      reveal_type(Model2.model_1) # N: Revealed type is "django.db.models.fields.related.ForeignObject[Any, Any]"
```

Comment:
If you also add a check for `Model2().model_1`, does it correctly show up as `Model1` type?

Reply:
No, it gets typed as `Any`. I will have a look to see if I can fix this.

Reply:
I've pushed a fix and updated the tests. It type-checks correctly now.